src/components/InputPanel.tsx

```diff
@@ -1,102 +1,171 @@
-import { useCallback, useContext, useEffect, useMemo, useState } from 'react';
-import InputContext, { INPUT_TYPES } from '../contexts/InputContext';
-import styles from '../css/RMLMappingEditor.module.scss';
-import CodeEditor from './CodeEditor';
-import { ReactComponent as PlusIcon } from '../images/plus.svg';
-import { ReactComponent as DownArrow } from '../images/down-arrow.svg';
+import { useCallback, useContext, useEffect, useMemo, useState } from "react";
```

Comment:
Our team prefers using single quotes. Can you revert these quote changes please?